src/pyfaf/storage/generic_table.py

```diff
@@ -43,6 +43,11 @@ def get_lob_path(self, name: str) -> str:
             if ex.errno != errno.EEXIST:
                 raise
 
+        for col in self.__table__._columns: #pylint: disable=no-member, protected-access
```

Comment:
I do not like this for-loop (because of performance). Can we instead have GenericsTableBase.get_lob_extension() which will return empty string. And in `Package` it will be redefined to return extension based on `pkgtype` attribute?